### PR TITLE
Closes #2020 chore: rename wt to weight in compute_egfr

### LINKED ISSUE #2020: General Issue: Update `wt` argument in `compute_egfr` to `weight`
### Background Information

AS part of our efforts to get to a `1.0` version of admiral, we are looking over all functions for consistency and user-friendliness.

Several functions use a `weight` argument, while the `compute_egfr` uses a `wt` argument for weight.

image

This should be updated to weight to be consistent.

### Definition of Done

- [ ] Function argument and description updated
- [ ] Deprecation process followed for arguments
- [ ] Tests, vignettes and templates updated as needed

### PR DESCRIPTION
**Pull request by StefanThoma**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/cran-release/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by StefanThoma** [comment]
@jeffreyad There seems to be an issue with the links from pkgdown as registered by our link-checker. How do I deal with this?

**Comment by bms63** [comment]
You can move the url to the `.lycheeignore` file 
image

**Comment by bms63** [comment]
I feel like those should have cran-release in the url, but that is another issue

**Comment by StefanThoma** [comment]
> I feel like those should have cran-release in the url, but that is another issue

Thanks!

File: docs/pkgdown.yml [hunk 1]
```diff
@@ -19,8 +19,8 @@ articles:
   queries_dataset: queries_dataset.html
   questionnaires: questionnaires.html
   visits_periods: visits_periods.html
-last_built: 2023-07-19T10:42Z
+last_built: 2023-08-03T06:39Z
 urls:
-  reference: https://pharmaverse.github.io/admiral/cran-release/reference/
-  article: https://pharmaverse.github.io/admiral/cran-release/articles/
+  reference: https://pharmaverse.github.io/admiral/reference
```
Comment (zdz2101) [inline docs/pkgdown.yml:24]: Think we need to keep the `"/cran-release/"` parts

**Comment by bms63** [comment]
Could you also change title of PR to what you are doing - Code Updated is pretty vague..  This becomes the commit message when merged in to `main/devel` now.  

image

**Comment by StefanThoma** [comment]
Thanks!

**Comment by jeffreyad** [comment]
@StefanThoma did you update the template `ad_adppk.R`?  I don't see it listed in files changed.

File: .lycheeignore [hunk 2]
```diff
@@ -3,3 +3,5 @@ https://github.com/pharmaverse/admiral/blob/main/
 https://github.com/pharmaverse/admiral/blob/main/inst/templates/ad_adxx.R
 irongut/CodeCoverageSummary@v1.2.0
 https://packagemanager.rstudio.com/cran/__linux__/focal/latest
+https://pharmaverse.github.io/admiral/reference
+https://pharmaverse.github.io/admiral/articles
```
Comment (bms63) [inline .lycheeignore:7]: Let's remove this from lychee please

File: docs/pkgdown.yml (hunk 1 above)
Comment (StefanThoma) [reply docs/pkgdown.yml:24]: Not sure how that happened!

File: .lycheeignore (hunk 2 above)
Comment (StefanThoma) [reply .lycheeignore]: Done

File: docs/pkgdown.yml
[the remark this answers is not in the document]
Comment (bms63) [reply docs/pkgdown.yml:1]: @StefanThoma can you put this in `.gitignore` please.

[the remark this answers is not in the document]
Comment (bms63) [reply docs/pkgdown.yml:1]: It looks like this was added in a long time ago - can just remove this line 
image

[the remark this answers is not in the document]
Comment (bundfussr) [reply docs/pkgdown.yml:1]: @StefanThoma , @bms63 , @cicdguy , `docs/pkgdown.yml` was added to create links across packages, e.g., if an example in admiralonco uses an admiral function it is linked automatically to the documentation of the admiral function.  It has stopped working when we introduced multiversions docs. Maybe we do not need it anymore if we find a different solution (see https://github.com/pharmaverse/admiralci/issues/131).

However, if we remove the file from `.gitignore` we should also remove the file itself.

**Pull request merged**